thavalon-server/src/lobby/lobby_impl.rs

```diff
@@ -245,15 +248,35 @@ impl Lobby {
         // the lobby is probably dead, so panic to blow up everything.
         if let Err(e) = self.database_game.start_game().await {
             log::error!("Error while starting game {}. {}", self.friend_code, e);
-            return LobbyResponse::Standard(Err(LobbyError::UnknownError));
+            panic!();
         }
 
         // Tell the players the game is about to start to move to the game page.
-        self.broadcast_message(&OutgoingMessage::StartGame).await;
-        self.status = DBGameStatus::InProgress;
+        self.broadcast_message(&OutgoingMessage::LobbyState(LobbyState::Game))
+            .await;
+        self.status = LobbyState::Game;
         let builder = self.builder.take().unwrap();
         builder.start();
-        return LobbyResponse::Standard(Ok(()));
+        LobbyResponse::None
+    }
+
+    /// Sends the current player list to the client.
+    async fn send_player_list(&mut self, client_id: String) -> LobbyResponse {
+        let mut client = self.clients.get_mut(&client_id).unwrap();
+        let player_list = self.builder.as_ref().unwrap().get_player_list().to_vec();
```

Comment:
This won't work after the game has started, right?

Reply:
Oh, we only call it if the game hasn't started yet

Reply:
Correct. After the game starts we have the players from the proposer list and it's fixed.